packages/selected-network-controller/src/SelectedNetworkController.ts

```diff
@@ -24,12 +26,14 @@ const getDefaultState = () => ({
 
 type Domain = string;
 
-const METAMASK_DOMAIN = 'metamask' as const;
+export const METAMASK_DOMAIN = 'metamask' as const;
 
 export const SelectedNetworkControllerActionTypes = {
   getState: `${controllerName}:getState` as const,
   getNetworkClientIdForDomain:
     `${controllerName}:getNetworkClientIdForDomain` as const,
+  getNetworkClientIdForMetamask:
+    `${controllerName}:getNetworkClientIdForMetamask` as const,
```

Comment:
should mention this is exposed as an action too in the future changelog

Reply:
No longer relevant as of https://github.com/MetaMask/core/pull/3908/commits/c78443841150f1f0acf3007fa9d67fae9bce5481